src/main/java/organice/logic/commands/ProcessingCommand.java

```diff
@@ -0,0 +1,88 @@
+package organice.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import organice.model.Model;
+import organice.model.person.Donor;
+import organice.model.person.Nric;
+import organice.model.person.Patient;
+import organice.model.person.Person;
+import organice.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Process a pair of donor and patient to provide a checklist to prepare for the organ transplant.
+ * Keyword matching is case insensitive.
+ */
+public class ProcessingCommand extends Command {
+
+    public static final String COMMAND_WORD = "processing";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Process the pair of donor and patient identified "
+            + "by their NRICs respectively and provide a checklist of the current status of the organ transplantation. \n"
+            + "Parameters: icP/PATIENT NRIC icD/DONOR NRIC \n"
+            + "Example: " + COMMAND_WORD + " icP/s4512345A icD/s7711123C";
+
+    public static final String MESSAGE_NOT_PROCESSED = "Donor or patient NRIC must be valid";
+
+    private String firstNRICString;
```

Comment:
I would suggest this approach:

A ProcessingCommand to store two Nric variables: `firstNric` and `secondNric`. 
The constructor of the ProcessingCommand to take in two Strings: `firstNricString` and `secondNricString`, then create two Nric objects like this:
```
public ProcessingCommand(String firstNricString, String secondNricString) {
        requireNonNull(firstNricString, secondNricString);
        firstNric = new Nric(firstNricString);
        secondNric = new Nric(secondNricString);
    }
```

Reply:
thank you and amended